stories/dem-info.stories.jsx

```diff
@@ -0,0 +1,15 @@
+import DemInfo from '../app/components/dem-info';
+import React from 'react';
+
+export default {
+    component: DemInfo,
+    args: {}
+}
+
+export const Primary = { args: {
+    user: {
+        dob: new Date('1990-10-20').toString(),
+        state: 'NY',
+        party: 'Independent'
+      }
+}}
```

Comment:
Please add a test case for when the user object is empty, and for when the user object doesn't exist.  These are edge conditions that could happen when the page is rendering but the data isn't fetched yet.

Reply:
Updated! Right now the fallback is just "N/A", but let me know if you have another idea...

Reply:
Please create one more case for `{args: {user: {}}` It's to deal with data fetch order issues.